pkg/features/kube_features.go

```diff
@@ -66,6 +66,13 @@ const (
 	// and also prevents them from being evicted from a node.
 	// Note: This feature is not supported for `BestEffort` pods.
 	ExperimentalCriticalPodAnnotation utilfeature.Feature = "ExperimentalCriticalPodAnnotation"
+
+	// owner: @davidopp
+	// alpha: v1.6
+	//
+	// Determines if affinity defined in annotations should be processed
+	// TODO: remove when alpha support for affinity is removed
+	AffinityInAnnotations utilfeature.Feature = "AffinityInAnnotations"
```

Comment:
Which-ever merges 1st the other will have to rebase, but it's definitely needed for this PR.